blazor/datepicker/date-format.md

```diff
@@ -9,24 +9,22 @@ documentation: ug
 
 #  Date Format in Blazor DatePicker Component
 
-Date format is a way of representing the date value in different string formats in text box.
+Date format is a way of representing the date value in different string formats in `DatePicker` control.
 
-By default, the DatePicker's format is based on the culture. You can also set the own custom format by using the [Format](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Calendars.SfDatePicker-1.html#Syncfusion_Blazor_Calendars_SfDatePicker_1_Format) property.
+The date format can be used to specify how the date value is displayed or entered in a `DatePicker` control
 
-> Once the date format property has been defined, it will be common for all the cultures.
+The string format of the date value specifies how the date value should be represented as a string. Different countries and regions have different conventions for representing the date value in a string format. For example, in the United States, the common date format is month/day/year (e.g., `12/28/2022`), while in many other countries, the day/month/year format is more common (e.g., `28/12/2022`). In addition to representing the date value in different string formats, it is also possible to specify the order in which the day, month, and year values appear in the string. For example, the day/month/year format could be written as `28-12-2022` or `28.12.2022`
```

Comment:
The string format of the date value specifies how the date value should be represented as a string. Different countries and regions have different conventions for representing the date value in a string format. In addition to representing the date value in different string formats, it is also possible to specify the order in which the day, month, and year values appear in the string. For example, the `day/month/year` format could be written as `28-12-2022` or `28.12.2022`